uiowa/uiowa
# update private_content to 2.1

Security notice: https://www.drupal.org/sa-contrib-2024-012

Changelog: https://git.drupalcode.org/project/private_content/-/compare/8.x-2.0...8.x-2.1?from_project_id=26575&straight=false

# To Test

```
ddev composer install && ddev blt ds --site=sppa.uiowa.edu
```

Check that private content is still private?
